pkg/db/postgres.go

```diff
@@ -194,14 +216,14 @@ func (pcf *postgresConnectionFactory) checkPostgresIsolationLevel() error {
 }
 
 func (pcf *postgresConnectionFactory) migrateDatabase() error {
-	logger := log.NewLogger(pcf.debug)
+	migrateLogger := newMigrateLogger(pcf.debug)
```

Comment:
As reminder after discuss: forgot to assign `m.Log = migrateLogger`

Reply:
Thanks! Applied 👍